src/renderers/common/BindGroup.js

```diff
@@ -2,11 +2,12 @@ let _id = 0;
 
 class BindGroup {
 
-	constructor( name = '', bindings = [], index = 0 ) {
+	constructor( name = '', bindings = [], index = 0, cacheKey ) {
```

Comment:
I think interessing complete `cacheKey|bindingsReference = []` to avoid undefined values.

Reply:
Done